pkgs/misc/mox-boot-builder/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, lib
+, fetchgit
+, buildPackages
+, pkgsCross
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mox-boot-builder";
+  version = "v2022.08.30";
+  src = fetchgit {
+    url = "https://gitlab.nic.cz/turris/mox-boot-builder.git";
+    rev = "0290b2cd9e14041ef7bcd267840f30d6c0a92ceb";
+    hash = "sha256-ZlbgJj/KgiDcF7Jl+wdNJi7iSivBIGF0W1qbrcIWBvk=";
+    fetchSubmodules = false;
+  };
```

Comment:
When given a gitlab source, we should use `fetchFromGitLab`. It uses the archive downloads rather than doing a more costly git clone. You can point to a non-`gitlab.com` host with the `domain` arg.

Note that this may (will?) change the hash, so make sure to update it as you would usually.